frontend/app/src/features/room/components/info/DifficultyBadge.tsx

```diff
@@ -0,0 +1,17 @@
+import { DifficultyType } from "@/constants/question"
+import { Badge } from "@chakra-ui/react"
+
+interface DifficultyBadgeProps {
+    difficulty: DifficultyType
+}
+
+export const DifficultyBadge = ({ difficulty }: DifficultyBadgeProps) => {
+
+    const colorSchemes: Record<DifficultyType, string> = {
+        "Easy": "green",
+        "Medium": "yellow",
+        "Hard": "red"
```

Comment:
```suggestion
        DIFFICULTY.EASY: "green",
        DIFFICULTY.MEDIUM: "yellow",
        DIFFICULTY.HARD: "red"
```

the enum object is defined to be used

Reply:
Resolved [dee539f](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g03/pull/18/commits/dee539f42bf5a8c51844ffde801e94933b1995dc)